apps/ex_nvr_web/priv/static/openapi.yaml

```diff
@@ -926,6 +1045,51 @@ components:
             vehicle_color:
               type: string
               description: The vehicle color.
+    RemoteStorage:
+      type: object
+      properties:
+        id:
+          type: number
+          description: RemoteStorage id
+        name:
+          type: string
+          description: RemoteStorage name
+        url:
+          type: string
+          description: RemoteStorage url
+        type:
+          type: string
+          enum: ["s3", "http"]
+          description: RemoteStorage's type (either s3 or http).
+    S3Config:
+      type: object
+      properties:
+        bucket:
+          type: string
+          description: RemoteStorage bucket name
```

Comment:
Update all the `RemoteStorage` to `Remote storage`. and use the specific storage type `S3 bucket name`.